std/socket.d

```diff
@@ -2839,7 +2839,7 @@ public:
                 newSocket._blocking = _blocking;                 //inherits blocking mode
             newSocket._family = _family;             //same family
         }
-        catch (Throwable o)
+        catch (Exception o)
```

Comment:
If you're going to muck with this one, then just make it a `scope(exit)`, since that's really what it should be.